### PR TITLE
Releases: 0.19.0

### PR DESCRIPTION
**Pull request by harding**
Todo:

- [x] Ensure ` _posts/en/posts/2019-11-09-release-0.19.0.md` contains the correct date in its file name
- [x] For ` _releases/0.19.0.md`, do:
    - [x] Ensure it contains the correct date in its YAML metadata
    - [x] Add magnet URI to YAML metadata (optional; can be done in a separate PR)

Note: tests are expected to fail because binaries have not yet been uploaded to `/bin/`.  When they're available, restart the tests and they should pass.

### CODE REVIEW

**Comment by harding** [comment]
Another note: as of this writing, BitcoinCore.org is not automatically updating.  @TheBlueMatt is working on getting access to investigate.  To tell whether or not the website has been fixed, look at [the homepage](https://bitcoincore.org/)---if the top story is about CVE-2017-18350, the site is (probably) building correctly; it if's about 0.18.1, it's almost certainly still broke.

File: _releases/0.19.0.md [hunk 1]
```diff
@@ -0,0 +1,1117 @@
...
+- Dan Gershony
+- Daniel Edgecumbe
+- Daniel Kraft
+- Daniel McNally
+- darosior
+- David A. Harding
+- David Reikher
+- Douglas Roark
+- Elichai Turkel
+- Emil
```
Comment (emilengler) [inline _releases/0.19.0.md]: I think I was added two times or it was another Emil

Comment (emilengler) [reply _releases/0.19.0.md]: Maybe this should belong to the actual bitcoin core repo release note

Comment (laanwj) [reply _releases/0.19.0.md]: Over-crediting isn't too bad anyway.  And yes, it's kind of too late to update this. There's been weeks in which you could look at and edit the release notes on the wiki.

Comment (emilengler) [reply _releases/0.19.0.md]: Ok it's my fault, will look at the release notes next time a bit deeper

**Comment by laanwj** [comment]
Updated the name and magnet link, and changed the actual release version to 0.19.0.1.

I wasn't really sure whether to update the announce version as well (after all, major release 0.19.0 *was* released, we just skipped the binaries for it), so kept that for 0.19.0 and added a note why the download is 0.19.0.1. Feel free to change this if you think it is unclear.

**Comment by harding** [comment]
Post-merge ACK.  The changes to .1 and the addition of the note to the blog post LGTM.  Thanks!

**Pull request merged**